Website/src/rendering/src/components/Speakers/SpeakersGrid.tsx

```diff
@@ -16,6 +22,13 @@ export type SpeakersGridProps = ComponentProps & {
 };
 
 const SpeakersGrid = (props: SpeakersGridProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasSpeakers = props.fields.data?.item;
```

Comment:
`hasSpeakers` is an object or `null` or `undefined` with that code. We could use `!!props.fields?.data?.item` for it to be a boolean and consume less memory.